SELearning.Infrastructure.Tests/CommentManagerTests.cs

```diff
@@ -8,148 +8,129 @@ namespace SELearning.Infrastructure.Tests
+        private static readonly Section section = new Section
+        {
+            Id = "1",
+            Title = "C#",
+            Description = "C# tools",
+            Content = new List<Content>()
+        };
+        private static readonly Content content = new Content
+        {
+            Author = "Sarah",
+            Section = section,
+            Id = 1,
+            Title = "Video on Entity Core",
+            Description = "Nice",
+            VideoLink = "www.hej.dk"
+        };
+        private IEnumerable<Comment> _comments = new List<Comment>()
+        {
+            new Comment { Author = "Amalie", Id = 1, Text = "Nice", Content = content, Rating = -10 },
+                new Comment { Author = "Albert", Id = 2, Text = "Cool but boring", Content = content },
+                new Comment { Author = "Paolo", Id = 3, Text = "This is a great video", Content = content },
+                new Comment { Author = "Rasmus", Id = 4, Text = "Very inappropriate", Content = content, Rating = 28 }
+        };
+        public CommentManagerTests()
+        {
+            //setting up the comment connection
+            var connection = new SqliteConnection("Filename=:memory:");
+            connection.Open();
+            var builder = new DbContextOptionsBuilder<CommentContext>();
+            builder.UseSqlite(connection);
+            CommentContext _context = new CommentContext(builder.Options);
+            _context.Database.EnsureCreated();
+
+            ICommentRepository _repo = new CommentRepository(_context);
+            _service = new CommentManager(_repo);
+
+            section.Content.Add(content);
+
+            _context.Comments.AddRange(
+                _comments
+            );
 
-        [Theory]
-        [InlineData("Amalie", "A really nice and professional video!")]
-        [InlineData("Y84Gmig", "Found this very confusing...")]
-        [InlineData("Anonymous user from ITU", "Cool explanation, but i didn't exactly get the part on regression testing. Can someone elaborate?")]
-        public void Post_given_acceptable_input_does_post(string author, string content)
+            _context.SaveChanges();
+        }
+
+        public void Post_given_acceptable_input_does_post()
         {
 
-            _service.PostComment(author, content);
-            //TODO: how do i check that this has actually been done?
         }
 
-        [Theory]
-        [InlineData("Amalie", "")]
-        [InlineData("Y84Gmig", "    ")]
-        [InlineData("", "")]
-        public void Post_given_empty_content_throws_exception(string author, string content)
+
+        public void Post_given_empty_content_throws_exception()
         {
-            _service.PostComment(author, content);
 
-            //TODO: how do i assert that it throws something and what do we want it to throw?
         }
 
         [Fact]
         public void Update_given_new_content_succeeds()
         {
-            //TODO: how to do this? 
+
         }
```

Comment:
Is it GitHub that shows this incorrectly or is this test empty? And the same for a test below.

Reply:
They are empty right now. Doing these last tests is the new thing on my todo. I just had to implement the methods in order for Joachim and me to merge, so this has not been 100% test driven development.